packages/react/README.md

```diff
@@ -101,3 +122,595 @@ function WidgetDeleter() {
   );
 }
 ```
+
+Find more examples in the **https://github.com/gadget-inc/examples** repo.
+
+### API Documentation
+
+`@gadgetinc/react` contains a variety of React hooks for working with your Gadget application's API from a React application. Specific code examples for your application's API can be found within your application's docs at https://docs.gadget.dev/api/<your-application-slug>
+
+### Setup
+
+Your React application must be wrapped in the `Provider` component from this library for the hooks to function properly. No other wrappers (like `urql`'s) are necessary.
+
+### `useFindOne(manager: ModelFinder, id: string, options: SingleFinderOptions = {}): [{data, fetching, error}, refetch]`
+
+`useFindOne` fetches one record from your Gadget database with a given `id`. `useFindOne(api.widget, 42)` is the React equivalent of `await api.widget.findOne(42)`. `useFindOne` expects a record with the given id to be found in the backend database, and will return an error in the `error` property of the result object if no record is found with the given `id`. `useFindOne` takes these options:
```

Comment:
When viewing the actual README, this paragraph is hard to get through. Starting sentences with `useFindOne` and ending with `id.` makes it really tough to read (the periods make it seem like we're accessing another property at first glance).

`id`. `useFindOne(api.widget, 42)`
`await api.widget.findOne(42)`. `useFindOne`
`id`. `useFindOne`

Just adding breaks might be the easiest way to get around this:

```suggestion
`useFindOne` fetches one record from your Gadget database with a given `id`.
`useFindOne(api.widget, 42)` is the React equivalent of `await api.widget.findOne(42)`.
`useFindOne` expects a record with the given id to be found in the backend database, and will return an error in the `error` property of the result object if no record is found with the given `id`.
`useFindOne` takes these options:
```